docs/contributing.md

```diff
@@ -1,16 +1,32 @@
 # Contributing
-Contributions are welcome. We have a few requests of any new contributors.
+Contributions are welcome. We have a few requests for new contributors:
 
-* Follow styling as designated in our [styling documentation](/docs/style.md).
-* Ensure that all changes which affect logic are covered by unit tests. 
-* Do not introduce any unit test failures/regressions.
-* Turn on automated github actions in your fork to have github run all the unit tests after pushing. See example below:
+* **Follow styling guidelines.**
+We have designated styling guideline to ensure uniformity and ease of communication. Please take a look at 
+the [styling documentation](/docs/style.md).
+
+* **Ensure that all changes are covered by unit tests.** 
+It is strongly recommended that unit tests are used to avoid regression and to ensure everything is still working.
```

Comment:
```suggestion
* **Ensure that all changes are covered by unit tests.** 
It is strongly recommended that unit tests are used to avoid regression and to ensure everything is still working.
```
Imo those two sentences are confusing when combined as they say different things about the same thing.
Maybe "Ensure that **critical** changes are covered by unit tests." Worlds currently can do whatever, basically, and so does javascript because we have no test framework for that.